docs/help-topics/stonesoup-import/stonesoup-import.yml

```diff
@@ -7,14 +7,13 @@
   tags:
   title: Add components
   content: |-
-    A “component” is an image that we build from source code in a repository. An application is a collection of these components working together. To start building your application, add a component by importing your code repository, or use one of our samples.
+    A “component” is a container image that we build from source code in a repository. An application is a collection of these components working together. To start building your application, add a component by importing your code repository, or use one of our samples.
 
     **Using code from your GitHub repository**
-    1. To use your own code, enter the link to your repository in the **Git repo URL** field. We verify your URL right away. You don’t even have to press **Enter**.
-    2. Click **Git options** to expand it and choose whether to make either of the following specifications:
-        1. Add a **Git reference** to point to code in a specific branch, tag, or commit that you want to use to build your application.
-        2. Indicate a **Context dir** to specify the subdirectory for the source code you want to use.
-    3. Click **Next**, then we scan and analyze your Git directory to determine how many components to add to your application. The types of files in your repository, and also your Git directory file hierarchy, determine the number and type of components to add. For example, we look for devfiles and Dockerfiles, and language-specific build configuration files like pom.xml for Java or package.json for Node.js.
+    1. To use your own code, enter the link to your repository in the **Git repository URL** field. We verify your URL right away. You don’t even have to press **Enter**.
```

Comment:
I could be overthinking this, but do we think it is confusing to tell the user to enter their link and then tell them they don't have to press enter? 

"To use your own code, enter the link to your repository in the **Git repository URL** field. We verify your URL as soon as it is entered, so you don't need to press anything." --> not sure if I like this either but brainstorming. WDYT @jseseCCS